pkg/windows/windows.go

```diff
@@ -191,12 +193,30 @@ func New(workerIgnitionEndpoint, vxlanPort string, instanceInfo *instance.Info,
 		return nil, errors.Wrapf(err, "unable to setup VM %s sshConnectivity", instanceInfo.Address)
 	}
 
+	// When in powershell, this will evaluate to an object containing information about the shell.
+	// When in cmd.exe, this will not evaluate to anything, and will be used literally. This can be used to determine
+	// if cmd or Powershell is being used.
+	powershellVersionTable := "($PSVersionTable)"
+	out, err := conn.run("echo " + powershellVersionTable)
+	if err != nil {
+		return nil, errors.Wrap(err, "error determining default shell")
+	}
+	var powershellDefault bool
+	if strings.TrimSpace(out) == powershellVersionTable {
+		powershellDefault = false
+	} else if strings.Contains(out, "PSEdition") {
```

Comment:
Is the presence of `PSEdition` good enough to claim powershell as default?  

Nit; if you are  looking after `PSEdition` you can fetch and compare the value using `$PSVersionTable.PSEdition`